test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -268,11 +280,14 @@ protected string GetBaselinePath(string subDir, string name)
                 return baselinePath;
 
             // check again for a platform specific dir
-            foreach (var baselineCOnfigDir in _baselineConfigDirs)
+            foreach (var baselineConfigDir in _baselineConfigDirs)
```

Comment:
As I mentioned above, this logic is almost a repetition of the above to handle Debug and Release cases. This can be merged with the logic above if we treat Debug and Release as part of the overall configuration. (But in a new PR. I think might complicate the selection order and we should discuss separately)